packages/node_modules/@cerebral/inferno/package.json

```diff
@@ -14,17 +14,21 @@
   },
   "homepage": "http://cerebral.github.io/cerebral-website",
   "devDependencies": {
-    "babel-plugin-inferno": "^3.2.0"
-  },
-  "dependencies": {
-    "cerebral": "next"
+    "babel-plugin-inferno": "^3.2.0",
+    "cerebral": "^3.6.0-1511263649330",
+    "function-tree": "^2.2.1",
+    "inferno": "^3.10.1",
+    "inferno-component": "^3.10.1",
+    "typescript": "^2.6.2"
   },
+  "dependencies": {},
```

Comment:
Dependencies should include `"cerebral": "next"`.

Dev-dependencies should not include `cerebral` or `function-tree` (this one comes along with `cerebral`).

No need to include `typescript` here, we keep in in the monorepo root and it makes it easier to update without having to go into each package changing the dependency.